semgrep/semgrep/semgrep_main.py

```
@@ -156,19 +156,17 @@ def main(
         logger.verbose(
             f"running {len(filtered_rules)} rules from {len(configs_obj.valid)} config{plural} {config_id_if_single} {invalid_msg}".strip()
         )
-
+        if len(errors) > 0:
```

Need a test plan, including to make sure that we aren't failing fast on targets

---

Actually it's probably worth adding a test case.